swift/swift.py

```diff
@@ -215,29 +178,13 @@ def destroyApp(self, name: str):
         app.deleteLater()
 
     @pyqtSlot(str, str)
-    def _routeToBus(self, busName: str, msg: str):
-        """Routes a signal from an app to the desired bus.
-
-        This is a slot for the broadcast signal of each app.
-
-        Args:
-            busName: A name of the desired bus that will transfer the signal.
-            msg: An input message to be transferred through the bus.
-        """
-        bus = self._buses[busName]
-        bus.write(msg)
-
-    @pyqtSlot(str)
-    def _routeToApp(self, msg: str):
-        """Routes a signal from a bus to the apps that subscribe to it.
-
-        This is a slot for the received signal of each bus.
+    def _broadcast(self, busName: str, msg: str):
+        """Broadcasts the message to the subscriber apps of the bus.
 
         Args:
-            msg: An input message transferred through the bus.
+            busName: Target bus name.
+            msg: Message to be broadcast.

```

Comment:
Oh, I thought that you knew this, because I've got to know this during reviewing your codes before.
As you can see [here](https://www.merriam-webster.com/words-at-play/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage), "cast" is an irregular verb.
Although "-casted" is also used, "-cast" seems the standard English.

Reply:
Wow.. I'm poor at English😂 You're right!